releasenotes/notes/43660.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
```

Comment:
I think we don't need a releasenote here. I wrote one before for the release: https://github.com/istio/istio/commit/0c8164f2dc0f7f405fd6036614ca8c11a4452f7e#diff-1208cdbb4e0a5998d314ac560f196977d73c184e22b9344ec4530410390fa09f, which resolved the same problem.